### ISSUE TITLE
Use Carla as a complement

### ISSUE THREAD

**Issue by Gabrielxd195** [LMMS/lmms#4634]
Carla is a tool that expands the compatibility of plugins in lmms. but it has two problems, it can not be automated or used as a complement. And about the latter, I came up with the idea that they put Carla as a selectable plugin in the lms plugins search engine. more or less it would look like this:
![complemento carla](https://user-images.githubusercontent.com/26127693/46239740-b24adb00-c36a-11e8-8bb3-ed6dfc9d6893.png)

LMMS is not compatible with lv2 plugins (something that baffles me), but with the integration of carla, this problem is solved. to mix audio calf lv2 plugins are perfect, but even using carla in lmms is impossible, the solution is to place carla as a plugin or plugin selectable in the plugin search, so you can use the lv2 plugin in the tracks as in the mixer. regards

**Comment by tresf** [comment]
> as a complement

As an effect?

**Comment by tresf** [comment]
BTW, Carla automation will be available in 1.3.0 if the PR gets merged: https://github.com/LMMS/lmms/pull/4571

**Comment by Gabrielxd195** [comment]
> > como complemento
> 
> Como un efecto?



> > as a complement
> 
> As an effect?

Ne exactly as an effect but rather as a complement, as a bridge to add effects, as seen in the image. if carla was in the effects list, it would be possible to add plugins that lmms does not officially recognize, use them in the music tracks or in the mixer, which solves (apparently) the incompatibility problems of plugins in lmms. although I do not know if it seems like a good idea or I do not know what the developers would say, but carla executes plugins almost all kinds of plugins, and it would be very useful.

**Comment by Gabrielxd195** [comment]
> > as a complement
> 
> As an effect?

Not as a simple effect, but as a complement of LMMS added to the list of effects, and with the possibility of adding it to any track, to use CARLA as a kind of bridge to execute lv2, dssi, sfz plugins and plugins that LMMS does not support currently.

**Comment by tresf** [comment]
Right, but functionally, as an effect.  Please update the title.

> LMMS is not compatible with lv2 plugins (something that baffles me)

We'd welcome the PR.  This software is just built from volunteers like you and me.

**Comment by Gabrielxd195** [comment]
> We'd welcome the PR. This software is just built from volunteers like you and me.

I would love to collaborate with you in LMMS, but unfortunately it is not scheduled, for now I just try to contribute ideas that improve LMMS.

**Comment by CYBERDEViLNL** [comment]
On the Carla repo there is also a issue for this: https://github.com/falkTX/Carla/issues/757 which is part of: https://github.com/falkTX/Carla/issues/873

**Comment by Lazerbeak12345** [comment]
https://github.com/LMMS/lmms/pull/4571 was closed in favor of https://github.com/LMMS/lmms/pull/5846, which was merged.

This makes more types of automation possible - though it's not yet possible to use Carla as an effect.

**Comment by mirk0dex** [comment]
There's VSTs out there that make Carla usable as an effect.

**Comment by mirk0dex** [comment]
Definitely. But we should focus on built-in plugin support instead of working on workarounds. Still it'd be nice to have Carla FX support.

**Comment by RainbowShatter** [comment]
We definetly need this. btw, Awesome!!
